### DISCUSSION TITLE
Container start but is in loop

### DISCUSSION THREAD

**Discussion by BeastGithub69** [haugene/docker-transmission-openvpn#2802] [General]
Hi,
2-3 days ago I realized that Sonar and I couldn't connect to Transmission anymore which was running for many months. After a few troubleshoot I decided to reinstall from scratch

My container starts but seems in a loop from what I'm seeing in the logs, that same message will loop the only change will be the temporary path. that home folder is empty and obviously can't access the UI

here's my log and config, hopefully, someone can help, I'm far from being a pro, and I'm more of a newbie enthusiast. 

[_Transmission-OpenVPN_logs.txt](https://github.com/haugene/docker-transmission-openvpn/files/14200710/_Transmission-OpenVPN_logs.txt)

docker run --cap-add=NET_ADMIN -d --net=bridge --name Transmission-OpenVPN 
-v /volume1/Download/Transmission-OpenVPN:/data 
-v /volume1/Docker/Transmission-OpenVPN:/config 
-v /volume1/Download/Transmission-OpenVPN/bin:/watch 
-v /volume1/Download/Transmission-OpenVPN\completed:/download 
-e TZ=Canada/Eastern 
-e OPENVPN_PROVIDER=PIA 
-e OPENVPN_CONFIG=ca_montreal 
-e OPENVPN_USERNAME=***** 
-e OPENVPN_PASSWORD=*****
-e PIA_OPENVPN_CONFIG_BUNDLE=openvpn
-e LOCAL_NETWORK=192.168.1.0/24 
-e TRANSMISSION_HOME=/volume1/Docker/Transmission-OpenVPN/config/transmission-home
-e TRANSMISSION_DOWNLOAD_DIR=/volume1/Download/Transmission-OpenVPN/completed 
-e TRANSMISSION_INCOMPLETE_DIR=/volume1/Download/Transmission-OpenVPN/incomplete 
-e TRANSMISSION_WEB_UI=transmission-web-control 
-e GLOBAL_APPLY_PERMISSIONS=false 
--log-driver json-file 
--log-opt max-size=10m 
-p 9091:9091 
--privileged=true 
--restart=always 
haugene/transmission-openvpn

**Comment by kosmonot** [comment]
-v /volume1/Download/Transmission-OpenVPN\completed:/download 

has a backslash instead of a forward slash.  Should be:

-v /volume1/Download/Transmission-OpenVPN/completed:/download

**Reply by pkishino** [reply]
Also,
TRANSMISSION_HOME=/volume1/Docker/Transmission-OpenVPN/config/transmission-home
Should be:
TRANSMISSION_HOME=/config/transmission-home
And the others as well..

**Reply by pkishino** [reply]
I would delete your transmission-settings file and remove these from the docker run
TRANSMISSION_HOME=/volume1/Docker/Transmission-OpenVPN/config/transmission-home
-e TRANSMISSION_DOWNLOAD_DIR=/volume1/Download/Transmission-OpenVPN/completed
-e TRANSMISSION_INCOMPLETE_DIR=/volume1/Download/Transmission-OpenVPN/incomplete

As they are set to wrong paths and you want to use default anyway..

**Reply by BeastGithub69** [reply]
After my correction yesterday it was still looping and when I came back today, everything was fine and even noticed a couple of downloads in Sonarr history.

Therefore despite my typo, I think the problem from the start was from the remote host. since it's started working on its own without really changing anything.

Thanks everyone for your help

**Comment by edgd1er** [comment]
My guess is that with backslash error, the container kept restarting and then triggered a PIA self protection measure.
you may wait few minutes/hours before retrying or mount a local dir with pia config and use custom config.

youmay start the container with DEBUG set, and/or change curl to curl -v to have more information about the process kill.

**Reply by BeastGithub69** [reply]
oops, 
I fix the forward slash but still same error, but still same error and same thing in the log event after enabling debugging
